### DISCUSSION TITLE
能否启用wlan直连

### DISCUSSION THREAD

**Discussion by star-o0O** [eiyooooo/Easycontrol_For_Car#45] [Ideas]
由于车机系统各种封闭
USB授权弹窗被屏蔽，有线连接无法使用，被迫使用无线连接
实际体验中，无论是手机热点还是车机热点，在1080p，90fps，12mbps参数下，均有目测1000ms+的延迟，触控体验较差
偶然发现，车机启用wlan直连后，由手机端通过wlan直连请求连接车机，连接后在上述参数下，目测延迟不超过200ms。（车机请求连接手机不行，延迟没有改善）
*有可能的原因：车机为固定IP，固定192.168.38.150，移动数据和wifi都要通过它，再与“车机”通信，感觉就像是套了一层网桥，这也导致了手机开热点给车机时，易控扫描不到手机

我的想法是易控启动时能否同时启用wlan直连，这样手机能搜到对等设备

**Comment by eiyooooo** [comment]
之前研究过一个星期wlan直连，需要手机端也安装客户端才能实现

目前易控的架构不需要安装在手机端安装软件，暂时没法支持wlan直连，可以使用第三方软件或者系统wlan设置里配对并建立wlan直连

估计下一个软件架构可以加上，但是目前没时间写😂

**Comment by eiyooooo** [comment]
[Nearby-v0.1-alpha-1.zip](https://github.com/user-attachments/files/16863397/Nearby-v0.1-alpha-1.zip) 解压后得到apk文件
可以测试一下，目前只有基础的建立wlan直连，后续添加功能
在两端安装，连接后再使用易控
